src/main/java/de/luhmer/owncloud/accountimporter/helper/API_SSO_Helper.java

```diff
@@ -0,0 +1,132 @@
+package de.luhmer.owncloud.accountimporter.helper;
+
+import java.io.InputStream;
+import java.lang.reflect.Type;
+
+import io.reactivex.Completable;
+import io.reactivex.functions.Action;
+import okhttp3.Request;
+import okhttp3.ResponseBody;
+import retrofit2.Call;
+import retrofit2.Callback;
+import retrofit2.Response;
+
+public class API_SSO_Helper {
```

Comment:
Hm, why classes names with underscores? o.O

Reply:
I'll rename and move this files :D It was just a quick proof of concept